documentation/slate/source/includes/nearby.md

```diff
@@ -1,42 +1,50 @@
-<a name="nearby"></a>Places nearby
+<a name="nearby"></a>![PLaces nearby](/images/places_nearby.png)Places nearby
 ==================================
 
+The Places Nearby feature displays the different transport options around a location - a GPS coordinate, 
+or address, for example. This function provides information like: stops, lines, timetables, 
```

Comment:
what do you mean by `  This function provides information like: stops, lines, timetables, next arrivals, etc` ?

not sure it's cristal clear